fem/tevaluator.hpp

```diff
@@ -54,8 +58,8 @@ class ShapeEvaluator_base<FE, IR, false, real_t>
 
    // default copy constructor
 
-   // Multi-component shape evaluation from DOFs to quadrature points.
-   // dof_layout is (DOF x NumComp) and qpt_layout is (NIP x NumComp).
+   /** Multi-component shape evaluation from DOFs to quadrature points.
```

Comment:
The `@brief` tag is missing from most of the `/** ... */`-style comments in this file.

Reply:
Yeah I pulled these ones in from another branch.  I'll add the brief tags.